### PR TITLE
NO-JIRA:e2e:document: how to run locally on HCP platform

### PR DESCRIPTION
**Pull request by Tal-or**
Documents all the the environment variables that are required for running e2e locally when NTO is running on Hypershift platform.

### CODE REVIEW

**Comment by Tal-or** [comment]
/hold
Depends on #1121

File: test/e2e/performanceprofile/functests/README.md [hunk 1]
```diff
@@ -22,4 +22,27 @@ DISCOVERY_MODE: to get an already deployed performanceProfile.
...
+### Running Tests Locally on Hypershift Platform
+
+When running e2e tests locally on a Hypershift platform, there are a couple of Environment variables 
+that should be set in advanced:
+
+* `CLUSTER_TYPE=hypershift` - A const that used to configure the e2e tests for running on Hypershift.
+
+
+* `CLUSTER_NAME` - Represents the name of the hosted-cluster. 
+When running locally,
```
Comment (jmencak) [inline test/e2e/performanceprofile/functests/README.md:36]: Nit: formatting, why line break so early?

File: test/e2e/performanceprofile/functests/README.md [hunk 2]
```diff
@@ -22,4 +22,27 @@ DISCOVERY_MODE: to get an already deployed performanceProfile.
...
+the name is usually the output of  `oc get hc <name-of-your-hosted-cluster> -n clusters --no-headers -o custom-columns=NAME:.metadata.name`
```
Comment (jmencak) [inline test/e2e/performanceprofile/functests/README.md:37]: Nit: s/output of  /output of /
i.e. extra space

Why "usually" the output of?  Why not always?

File: test/e2e/performanceprofile/functests/README.md [hunk 3]
```diff
@@ -22,4 +22,27 @@ DISCOVERY_MODE: to get an already deployed performanceProfile.
...
+
+
+* `HYPERSHIFT_MANAGEMENT_CLUSTER_KUBECONFIG` - The path for the Kubeconfig file of the management/hub cluster.
+
+
+* `KUBECONFIG` - The path for the Kubeconfig file of the hosted cluster.
```
Comment (jmencak) [inline test/e2e/performanceprofile/functests/README.md:43]: Why do we capitalize kubeconfig? Similarly elsewhere.

File: test/e2e/performanceprofile/functests/README.md [hunk 4]
```diff
@@ -22,4 +22,27 @@ DISCOVERY_MODE: to get an already deployed performanceProfile.
...
+
+
+* `HYPERSHIFT_HOSTED_CONTROL_PLANE_NAMESPACE` - The name of the namespace where all the control-plane components reside.
```
Comment (jmencak) [inline test/e2e/performanceprofile/functests/README.md:46]: I strongly suggest replacing:
```
<key> - <key_explanation>
```
in favour of
```
<key>: <key_explanation>
```
Note there is no space between `key` and `:`.  This is also what OpenShift docs seem to be using.  Using a hyphen for this is not a correct usage in my view.

**Review by jmencak — COMMENTED** [review]
Thank you for the docs!

File: test/e2e/performanceprofile/functests/README.md [hunk 5]
```diff
@@ -22,4 +22,27 @@ DISCOVERY_MODE: to get an already deployed performanceProfile.
...
-RESERVED_CPU_SET, ISOLATED_CPU_SET, OFFLINED_CPU_SET: strings that present the CPU sets distributed between reserved, isolated, and offlined CPU profile specifications. The runner is responsible for validating that these values are compatible with the testing environment. 
\ No newline at end of file
+RESERVED_CPU_SET, ISOLATED_CPU_SET, OFFLINED_CPU_SET: strings that present the CPU sets distributed between reserved, isolated, and offlined CPU profile specifications. The runner is responsible for validating that these values are compatible with the testing environment.
+
+### Running Tests Locally on Hypershift Platform
+
+When running e2e tests locally on a Hypershift platform, there are a couple of Environment variables 
+that should be set in advanced:
+
+* `CLUSTER_TYPE=hypershift` - A const that used to configure the e2e tests for running on Hypershift.
```
[the remark this answers is not in the document]
Comment (Tal-or) [reply test/e2e/performanceprofile/functests/README.md:32]: @jmencak not necessarily. 
In the link you referred the environment variable is being set as part of the deployment.
But when running the e2e tests locally you're not running inside NTO pod so noone is setting this environment variable.

**Comment by jmencak** [comment]
Thank you for the changes, looks better now.
/lgtm
/approve

**Pull request merged**